pkg/rbac/sharedData.go

```diff
@@ -81,6 +87,11 @@ func (cache *Cache) PopulateSharedCache(ctx context.Context) error {
 
 }
 
+func (cache *Cache) SharedCacheDisabledClustersValid() bool {
```

Comment:
Make private (lowercase)
```suggestion
func (cache *Cache) sharedCacheDisabledClustersValid() bool {
```